src/index.js

```diff
@@ -1,11 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react";
+import ReactDOM from "react-dom";
+import "bootstrap/dist/css/bootstrap.css";
```

Comment:
is this still needed?

Reply:
I do not think so, since I have removed its in .json I will remove it and see if anything breaks.

Reply:
cool, thanks